src/components/results/results.tsx

```diff
@@ -1,23 +1,27 @@
 import { type FC, useEffect, useState } from "react";
-import { Button, Stack, Typography } from "@mui/material";
+import { Stack } from "@mui/material";
 import { useRouter } from "next/router.js";
 import { signIn, useSession } from "next-auth/react";
 
 import { pages } from "~/router.js";
 import { useBBWallet } from "~/service/bb-wallet.js";
 import { api } from "~/utils/api.js";
-import { isAnonymousSession, missingUsername } from "~/utils/session.js";
 
-import { styles } from "./styles.js";
+import { type AchievementId } from "~/types/index.js";
+import { Score } from "~/components/score/index.js";
+import { isAnonymousSession, missingUsername } from "~/utils/session";
```

Comment:
Missing file extension? 
We may need an extra linter rule to catch these things.